backend/api/search.py

```diff
@@ -84,34 +84,51 @@ def search_avanced():
 
     try:
         search_string = query['searchString']
-        authors = query.get('authors', [])
-        tags = query.get('tags', [])
+        authors = query.get('authors')
+        tags = query.get('tags')
         status = query.get('status')
         date_from = query.get('date_from')
         date_to = query.get('date_to')
         description = query.get('description')
+        title = query.get('')
     except KeyError:
```

Comment:
But I do exception handling for searchString, because that is actually required, as written.